distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/restito/ChunkedContent.java

```diff
@@ -44,25 +47,14 @@
      *
      * @return the minimum headers required for sending a chunked data response
      */
-    private static Action getHeaders() {
+    public static Action getChunkedResponseHeaders() {
```

Comment:
I made them public so they can be used as a static utility method for getting generic range/chunked headers.